java/ql/lib/semmle/code/java/dataflow/RangeAnalysis.qll

```diff
@@ -292,7 +295,8 @@ module SignInp implements SignAnalysisSig<Sem> {
 module Modulus implements ModulusAnalysisSig<Sem> {
   class ModBound = Bound;
 
-  predicate semExprModulus = exprModulus/4;
```

Comment:
Does this need deprecation?

Reply:
I don't really consider modulus analysis to be a very public api, so I'd lean towards no - and that's for C++. On the Java side this is part on an on-going refactor, and the temporary `semExprModulus` was only temporarily available in a very short window. So on the Java side, I'd say definitely not.